sdk/identity/identity/integration/AzureKubernetes/index.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const { BlobServiceClient } = require("@azure/storage-blob");
+const { ManagedIdentityCredential } = require("@azure/identity");
+
+async function main() {
+  const account2 = process.env.IDENTITY_STORAGE_NAME_2;
```

Comment:
Script looks good to me. Just one small thing, we can just call it 'account' since there's only one account?